service/googlechat/googlechat.go

```diff
@@ -0,0 +1,97 @@
+// Package googlechat provides message notification integration sent to multiple
+// spaces within a Google Chat Application.
+package googlechat
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/pkg/errors"
+	"google.golang.org/api/chat/v1"
+	"google.golang.org/api/googleapi"
+	"google.golang.org/api/option"
+)
+
+//go:generate mockery --name=spacesMessageCreator --output=. --case=underscore --inpackage
+type spacesMessageCreator interface {
+	Create(string, *chat.Message) createCall
+}
+
+//go:generate mockery --name=createCall --output=. --case=underscore --inpackage
+type createCall interface {
+	Do(...googleapi.CallOption) (*chat.Message, error)
+}
+
+// Compile-time check to ensure that client implements the spaces message service
+// interface.
+var (
+	_ spacesMessageCreator = new(messageCreator)
+	// Compile-time check to ensure that client implements the create call interface.
+	_ createCall = new(chat.SpacesMessagesCreateCall)
+)
```

Comment:
```suggestion
var (
	// Compile-time check to ensure that client implements the spaces message service
	// interface.
	_ spacesMessageCreator = new(messageCreator)
	// Compile-time check to ensure that client implements the create call interface.
	_ createCall = new(chat.SpacesMessagesCreateCall)
)
```

Small formatting change.